app/components/article-featured-video.js

```diff
@@ -75,20 +71,22 @@ export default Component.extend(RenderComponentMixin, JWPlayerMixin, {
 	},
 
 	willDestroyElement() {
-		$(window).off('scroll', this.onScrollHandler);
+		document.body.classList.remove('featured-video-on-scroll-active');
+		window.removeEventListener('scroll', this.throttleOnScroll);
 	},
 
 	actions: {
-		/**
-		 * FIXME FEATURED VIDEO A/B TEST ONLY
-		 */
 		dismissPlayer() {
-			this.$().removeClass(scrollClassName).addClass('is-dismissed');
+			this.set('isOnScrollActive', false);
+			this.set('isOnScrollClosed', true);
+			this.triggerOnScrollStateChange('closed');
 
-			this.player.setMute(true);
-			this.track(trackActions.click, 'onscroll-close');
+			if (this.player) {
+				this.player.setMute(true);
+			}
+			document.body.classList.remove('featured-video-on-scroll-active');
```

Comment:
How about defining a property for this classname, so that we can reuse across the component?